lib/components/artist/related_artists/index.js

```diff
@@ -3,25 +3,36 @@
 
 import Relay from 'react-relay';
 import React from 'react';
-import { ScrollView, StyleSheet, View } from 'react-native';
+import { ScrollView, StyleSheet, View, Dimensions } from 'react-native';
 
 import colors from '../../../../data/colors';
 import SerifText from '../../text/serif';
 import ImageView from '../../opaque_image_view';
 import RelatedArtist from './related_artist'
 
+const isPhone = Dimensions.get('window').width < 700;
+
 class RelatedArtists extends React.Component {
   render() {
-    const artists = this.props.artists
+    const artists = this.props.artists;
     return (
       <View style={styles.container}>
         <SerifText style={styles.heading}>Related Artists</SerifText>
         <View style={styles.artistContainer}>
-          { artists.map(artist => <RelatedArtist key={artist.id} artist={artist} /> ) }
+          { this.renderArtists() }
         </View>
       </View>
     );
   }
+
+  renderArtists() {
+    const artists = this.props.artists;
```

Comment:
@mennenia this is just an illustrative commit to show the spacer idea - how do you feel about this? It fixes the issue for iPhone: 
![simulator screen shot may 31 2016 5 23 13 pm](https://cloud.githubusercontent.com/assets/2712962/15694754/487fcdce-2755-11e6-8eef-94e9f6fee4df.png)

but not for iPad, so we'll have to do something else for that:

![simulator screen shot may 31 2016 5 24 59 pm](https://cloud.githubusercontent.com/assets/2712962/15694760/5b674c1e-2755-11e6-878d-d146cb3f7ac2.png)